db/migrate/20161224170703_create_groups.rb

```diff
@@ -0,0 +1,15 @@
+class CreateGroups < ActiveRecord::Migration[5.0]
+  def change
+    create_table :groups do |t|
+      t.integer :size, null: false, default: 1
+      t.integer :status, null: false, default: 0
+      t.belongs_to :leader, index: true, null: false
```

Comment:
Yeah, I was thinking that the leader would always be a student because they eventually get the permission to select a suite for the group.